packages/babel-core/src/transformation/file/file.ts

```diff
@@ -44,15 +44,15 @@ export type NodeLocation = {
 };
 
 export default class File {
-  _map: Map<any, any> = new Map();
-  opts: any;
-  declarations: any = {};
-  path: NodePath<t.Program> = null;
-  ast: any = {};
+  _map: Map<unknown, unknown> = new Map();
+  opts: { [key: string]: any };
+  declarations: { [key: string]: t.Identifier } = {};
+  path: NodePath<t.Program>;
+  ast: t.File;
   scope: Scope;
-  metadata: {} = {};
+  metadata: { [key: string]: any } = {};
   code: string = "";
-  inputMap: any | null = null;
+  inputMap: any;
```

Comment:
Is it `SourceMapConverter` from "convert-source-map"?

Reply:
It seems yes, but there are such definitions in many files.
`type SourceMap = any;`
I don't know what the reason is.

`packages\babel-core\src\transformation\file\generate.ts`
`packages\babel-core\src\transformation\index.ts`
`packages\babel-core\src\transformation\file\merge-map.ts`